package.json

```
@@ -33,16 +33,19 @@
     "build-ts": "rm -rf ./dist && tsc && rsync -a --include '*/' --include '*.d.ts' --exclude '*' ./typechain ./dist/",
     "copy-idl": "mkdir -p dist/src/svm/assets/idl && cp src/svm/assets/idl/*.json dist/src/svm/assets/idl/",
     "build": "yarn build-evm && yarn build-svm && yarn generate-svm-assets && yarn build-ts && yarn copy-idl",
-    "test-evm": "IS_TEST=true hardhat test",
+    "test-evm": "yarn test-evm-hardhat && yarn test-evm-foundry",
+    "test-evm-hardhat": "IS_TEST=true hardhat test",
+    "test-evm-foundry": "FOUNDRY_PROFILE=local forge test",
     "test-svm": "anchor test -- --features test",
     "test": "yarn test-evm && yarn test-svm",
     "test:report-gas": "IS_TEST=true REPORT_GAS=true hardhat test",
+    "evm-contract-sizes": "sh scripts/evm-contract-sizes.sh",
     "generate-evm-assets": "rm -rf typechain && TYPECHAIN=ethers yarn hardhat typechain",
     "prepublish": "yarn build && hardhat export --export-all ./cache/massExport.json && ts-node ./scripts/processHardhatExport.ts && prettier --write ./deployments/deployments.json && yarn generate-evm-assets",
     "pre-commit-hook": "sh scripts/preCommitHook.sh"
   },
   "dependencies": {
-    "@across-protocol/constants": "^3.1.37",
+    "@across-protocol/constants": "3.1.43",
```

```suggestion
    "@across-protocol/constants": "^3.1.43",
```

why did you remove the ^?

---

Ah, it's because of the command I called `yarn add @across-protocol/constants@3.1.43`. Should have done this instead `yarn add @across-protocol/constants@^3.1.43`